src/main/java/org/osc/controller/nsc/entities/InspectionHookNSCEntity.java

```diff
@@ -0,0 +1,79 @@
+package org.osc.controller.nsc.entities;
+
+import static javax.persistence.FetchType.LAZY;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+
+@Entity
+@Table(name="InspectionHook")
+public class InspectionHookNSCEntity {
```

Comment:
For consistency with the sample plugin please remove the `NSC` from the entity names, it should be simply InspectionHookEntity